pkg/frameworkPackage.targets

```diff
@@ -43,6 +43,15 @@
       <Version>$(PlatformPackageVersion)</Version>
       <TargetFramework>$(TargetFramework)</TargetFramework>
     </Dependency>
+
+    <!-- Include the update to v1 lineup package, which now has no runtime package mappings.
+         This will reduce the number of packages downloaded when mixing v1 and later
+         packages as well as reduce our dependence on conflict resolution for dropping
+         those v1 runtime packages -->
+    <Dependency Condition="'$(IsLineupPackage)' == 'true'" Include="Microsoft.NETCore.Targets">
+      <Version>2.0.0</Version>
```

Comment:
> reason to not be a ProjectReference
It was what we were doing for platforms and it reduces a bit of build time.  I thought about this and another reason was that I didn't expect us to ever update this dependency.  In fact, we'll likely delete the package authoring after 2.0.

> reason to reference this in the private transport packages
I'm just using that to flow the dependency, we could instead add the reference to the package in core-setup if we wanted.